test/xpu_api/utilities/pairs/pairs.pair/piecewise_g.pass.cpp

```diff
@@ -0,0 +1,144 @@
+// -*- C++ -*-
```

Comment:
I think this test is also redundant. `piecewise_construct.pass.cpp` would be enough.

Reply:
Done: test deleted